cpp/src/utilities/cusparse_helper.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 #include <cusparse.h>
+#include <rmm/device_buffer.hpp>
 #include "rmm_utils.h"
```

Comment:
Should rmm_utils.h be removed? 
Seems like `./cpp/include/rmm_utils.h:37:#define ALLOC_TRY(ptr, sz, stream)` should go away

Reply:
If we do so we can get rid of this too: https://github.com/kaatish/cugraph/blob/a0699aa3f8863df00ca30cf7e39da52d0c92ea52/cpp/src/utilities/error_utils.h#L113

Reply:
rmm_utils.h is used everywhere just to include error_utils.h, rmm.h and thrust_rmm_allocator.h. We could remove this file, but it would mean adding all those three everywhere else. What would you suggest?

Reply:
Reducing the number of `#include`  may not be a strong enough motivation to keep  `rmm_util.hpp`. If we start going down that path the risk is that we keep adding header files to encapsulate other `#include`. What do you think?

Reply:
Sounds good. Removed `rmm_utils.h`